cmd/heartbeats/main.go

```diff
@@ -122,7 +134,28 @@ func main() {
 		log.Fatalf("Failed to initialize tracing: %v", err)
 	}
 	defer tracer.Shutdown(ctx)
-	c, err := client.NewClientHTTP([]cehttp.Option{cloudevents.WithTarget(sink)}, nil)
+
+	opts := make([]cehttp.Option, 1)
```

Comment:
My suggestion caused the test failure
```suggestion
	opts := make([]cehttp.Option, 0, 1)
```